python/composio/tools/toolset.py

```diff
@@ -1317,6 +1317,8 @@ def initiate_connection(
         entity_id: t.Optional[str] = None,
         redirect_url: t.Optional[str] = None,
         connected_account_params: t.Optional[t.Dict] = None,
+        *,
+        auth_scheme: t.Optional[str] = None,
```

Comment:
Consider adding validation for auth_scheme values to ensure only valid authentication schemes are passed. This could prevent issues where typos or invalid schemes silently fail to match existing integrations.